lua/wikis/commons/ThisDay.lua

```diff
@@ -0,0 +1,118 @@
+---
```

Comment:
should we maybe make this a widget (`Widget/ThisDay.lua`)

would allow to easily replace the old stuff as the new stuff will be in sep module
(can kick the old one after its usage was replaced)